### DISCUSSION TITLE
Support Buildkit 0.12.0

### DISCUSSION THREAD

**Discussion by ryanneilyoung** [docker/buildx#1942] [Q&A]
### Description

Hi,

My team has been wanting to handle better caching in our Dockerfiles, especially around multi-stage builds. We've found something that works, but the new cache changes in BuildKit should allow us to use the ` image-manifest=true` option and post our builds to AWS ECR much easier, and more efficiently.

**Comment by ryanneilyoung** [comment]
for now it looks like I can specify the buildkit version when creating the builder:

docker buildx create --use --driver=docker-container --name=test --bootstrap --driver-opt=image=moby/buildkit:latest

and the cache-to supports the image-manifest arg, so things are working. 

It would be nice to have 0.12 as the default buildkit version for the builder, but for now I'm excited about how well this works!

**Reply by mbentley** [reply]
Thanks for pointing out that you can use`image-manifest=true` as an argument in `cache-to`.  In testing out the `3.0.0-alpha.1` distribution release, I found that this was required for my caching to work otherwise it was erroring out with errors like this (these aren't correlated errors from the buildx command and the registry as it was from two different builds but the general error is the same/correct):

```
#11 ERROR: error writing manifest blob: failed commit on ref "sha256:73e98dd990b3b49551be1c7218a2bd7d432cbeb479e9a56e55379b8ad1b2104d": unexpected status from PUT request to https://registry.casa.mbentley.net/v2/mbentley/buildkit/manifests/v0.12.0-amd64-cache: 400 Bad Request
```

```
time="2024-01-31T22:10:10.110032234Z" level=error msg="response completed with error" err.code="manifest blob unknown" err.detail="sha256:4eaa9aede0fb78345c8c61e1145d0a071bb922540d81be464818dde565513895" err.message="blob unknown to registry" go.version=go1.21.5 http.request.contenttype=application/vnd.oci.image.index.v1+json http.request.host=registry.casa.mbentley.net http.request.id=06d01c26-e2fd-460b-b616-0bb060ad3f19 http.request.method=PUT http.request.remoteaddr="192.168.2.171:45784" http.request.uri=/v2/mbentley/buildkit-test-regcache/manifests/sb8200_stats-cache http.request.useragent=buildkit/v0.12 http.response.contenttype=application/json http.response.duration=1.928394ms http.response.status=400 http.response.written=1849 instance.id=41a8f0cf-d5a9-4de0-854a-ec37704a154d service=registry vars.name=mbentley/buildkit-test-regcache vars.reference=sb8200_stats-cache version=3.0.0-alpha.1
```

**Comment by crazy-max** [comment]
> It would be nice to have 0.12 as the default buildkit version for the builder, but for now I'm excited about how well this works!

0.12.0 has been published few days ago and we usually wait a week before updating `buildx-stable-1` tag.
